### DISCUSSION TITLE
在 style 里面用 v-bind('token.marginXXS') 如何带上 px 的单位？

### DISCUSSION THREAD

**Discussion by nd-lqj** [vueComponent/ant-design-vue#7111] [Q&A]
在 scss 类型的 style 里面需要使用 token 的变量值，如果是使用颜色值或者其他字符串类型的都是OK的，但是在使用数值类型的变量的时候，要如何给这个数值带上单位px呢？

**Comment by ryne6** [comment]
font-size: v-bind(titleSize + 'px');

**Reply by nd-lqj** [reply]
这么处理的话，就需要对用到的token值单独设置变量存储，就感觉很不优雅。

**Reply by hzlzsm** [reply]
> 这么处理的话，就需要对用到的token值单独设置变量存储，就感觉很不优雅。

margin:calc(1px * v-bind('token.margin'))
